src/dispatch/incident/views.py

```diff
@@ -139,6 +140,21 @@ def create_incident(
     return incident
 
 
+@router.post(
+    "/{incident_id}/resources",
+    response_model=IncidentUpdate,
+    summary="Creates resources for an existing incident.",
+)
+def create_incident_resources(
```

Comment:
Should we do this as a background task with background_tasks.add_task()?

Reply:
ditto

Reply:
Done.